strawberry/aiohttp/handlers/http_handler.py

```diff
@@ -37,7 +37,7 @@ async def handle(self) -> web.StreamResponse:
     async def get(self, request: web.Request) -> web.StreamResponse:
         if self.should_render_graphiql(request):
             return self.render_graphiql()
-        return web.HTTPNotFound()
+        raise web.HTTPNotFound()
```

Comment:
Returning http exceptions has been deprecated.

Reply:
should we make a patch release for this?

Reply:
i don't think thats necessary. this change doesn't really affect users of the library.

Reply:
mostly because we are going to do a release before the deprecated thing gets removed? :)